### PR TITLE
V1.0.0 - Beta

### PR DESCRIPTION
**Pull request by A-Jammoul**
- Removed unused msg folder

- Update configuration files. Removed unused bridge.yaml file

- Updated README

- Moved src files into new vda5050_connector

- Updated includes in header files

- Updated comments, removed usage of namespaces in header files

- Removed unused header object for Daemons.

- Created new Utils source file

- Updated state message in AGV and state mockups

- Updated tests

- Updated CMakeLists

- Removed unused launch files (Missing configuration files)

- Update knotvector type

- Removed logs and exit

- Rename the whole thing
  
- Updated the names of the nodes for more clarity. Removed the usage of the word daemon.

- Updated cmake lists

- Split configuration of nodes. And merged mqtt configuration into one file.

- Merged all nodes into one launch file. Added config file for each node.

- Updated tests

- Updated documentation

- Removed the connection publisher node and its documentation

- Updated the state aggregator to send connection state messages

- Updated documentation

- Updated launch files

- Added support for Ubuntu 16

- Updated configuration

- Updated mockups

- Added support for default ros messages for battery

- Added rate and updated subscribed channel for velocity

- Updated documentation

- Added how to run state mockup to documentation

- Updated timestamp field to timeStamp

- Added zone update connection

- Update name of instantAction topic

- Transformed all channels and parameters to snake case 

- Updated agvposition to pose 

- Updated documentation for the channels 

- Fixed bug in setting the operating mode of the device. 

- Added operating modes to the state mockup. 

- Added check while setting errors to avoid having a duplicate error type. 

- Removed usage of namespace std in the vda5050_connector. 

- Standardized parameters to be snake_case 

- Fixed bug in SetOperatingMode to set the operating mode properly 

- Added internal errors that expire after 10 seconds 

- Updated Contribution.md 

- Fixed bug with including deleted messages 

- Updated comments 

- Removed unused ros publishers 

- Updated the state mockup 

- Updated the action client after deleting ros messages

### CODE REVIEW

**Review by JadKeryakos — APPROVED** [review]

File: include/vda5050_connector/order_manager.h
Comment (frothm) [inline include/vda5050_connector/order_manager.h:1]: Same here, not sure if Doxygen will process the `// ...` comments related to variables.

File: include/vda5050_connector/vda5050node.h
Comment (frothm) [inline include/vda5050_connector/vda5050node.h:1]: Commenting of variables should be Doxygen-compatible again

File: include/vda5050_connector/state_aggregator.h
Comment (frothm) [inline include/vda5050_connector/state_aggregator.h:1]: Variable documentation should be Doxygen compatible.

**Review by frothm — CHANGES_REQUESTED** [review]
In 4 files (see comments), the commenting of variables is not Doxygen-conform. Please change these one-line comment blocks to a format that is recognized by Doxygen. E.g. `//! ...` would work, or the commentary after the variable definition with `/*< ... */`. Reference: [Doxygen documentation](https://www.doxygen.nl/manual/docblocks.html).
Thanks a lot and best regards :)

**Pull request merged**